libraries/lib-music-information-retrieval/tests/TatumQuantizationFitBenchmarkingOutput/summary.txt

```diff
@@ -0,0 +1,5 @@
+AUC: 0.9121725731895223
+Strict Threshold (Minutes-and-Seconds): 0.8367879995455243
+Lenient Threshold (Beats-and-Measures): 0.7336582045210642
+Octave error RMS: 0.1248861169665804
+Audio file hash: -1840.31005859375
```

Comment:
From @Paul-Licameli:
> That's strange -- I thought the checksum would be a summation of samples of audio files, which would tend to average toward 0.

Reply:
Looks ok. I've just run the benchmarking again, counting the total number of samples, which is 84540711. To get the average DC offset in dB, `20*log10(1840.31005859375/84540711) == -93.24349792273436`, close to the noise floor for 16bit audio.